src/ovs/aca_ovs_l2_programmer.cpp

```diff
@@ -189,10 +189,6 @@ int ACA_OVS_L2_Programmer::configure_port(const string vpc_id, const string port
     throw std::invalid_argument("tunnel_id is 0");
   }
 
-  if (overall_rc != EXIT_SUCCESS) {
-    throw std::runtime_error("Invalid environment with br-int and br-tun");
```

Comment:
Is this validation unused any more?

Reply:
correct, it is leftover from the previous logic.